oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -122,20 +123,22 @@ Frontend Technology Selection
 
    **Rationale**: edX JavaScript should be written consistent with the latest
    ECMA-262 specification in order to ensure future support, the largest
-   community and the availability of modern features. Currently, the ECMA-262
-   edition 6 (ES6) is the edition with the largest support and should be used.
-   To support this syntax in older browsers, use `Babel`_. Babel may also be
-   configured to add syntax extensions widely adopted by the community of our
-   recommended framework (e.g., `JSX`_).
+   community and the availability of modern features. Currently, ES2022 is
+   the latest version with full compatibility from `the browsers we support`_.
+   There is no need to use Babel, core-js, nor polyfills to provide backwards
+   compatibility with unsupported browsers nor to transform ES2022 code into
+   ES6/ES5 code.
 
-   **Exception**: Much of edX's existing front end code is written conformant
-   to the edition of ECMA-262 released in 2009 (ES5). Files written in ES5
-   should be gradually converted to the newer standard as new development in
-   those feature areas requires.
+   **TypeScript**: TypeScript should be utilized in frontend code wherever it
+   seems to provide value (especially in any code that is reused: libraries, API
+   wrappers, shared components, data type definitions, etc.).
```

Comment:
Would it be worth linking to https://docs.openedx.org/projects/openedx-proposals/en/latest/best-practices/oep-0067/decisions/frontend/0008-typescript.html here?